tests/scala/toolchain/test

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+. "$(dirname "$0")"/../../common.sh
+
+
+bazel_cquery_output="$(
+    bazel cquery 'deps(:parent1 + :parent2, 1)' --output graph --nograph:factored |&
+        sed -nr 's/^ *"\/\/scala\/toolchain:child \(([0-9a-f]*)\)"$/\1/p'
```

Comment:
It would be helpful if you could add a comment here describing what you're checking for, so folks don't have to spend the mental energy to decode the sed command unless they really need to.

Reply:
Good idea :sweat_smile: